package.json

```diff
@@ -16,7 +16,7 @@
   },
   "scripts": {
     "test-server": "node_modules/mocha/bin/mocha test/server",
-    "test-contract": "truffle test test/contracts",
+    "test-contract": "truffle test",
```

Comment:
since truffle is actually mocha we can remove `test-contract` and `test-srver` and just have `test`